windows_containers/configuring-byoh-windows-instance.adoc

```diff
@@ -0,0 +1,65 @@
+[id="configuring-byoh-windows-instance"]
+= Configuring Bring-Your-Own-Host (BYOH) Windows instances
+
+-NEED AN INTRO-
+
+== Prerequisites
```

Comment:
This prereqs header needs a preceding unique ID, such as: `[id="prerequisites_configuring-byoh-windows-instance"]`

Reply:
Also, a procedure must be in the form of a module nested within an assembly. So you might want to reconsider hosting these prereqs in the module, not at the assembly level. They would only make sense at the assembly level if they apply entirely to _all_ procedures in the assembly. Not sure they do in this case. If they're specific to just configuring, then it can be formatted this way (and without a unique ID as I mentioned in the previous comment):

> `.Prerequisites`